pkg/controller/cassandracluster/reconcile_test.go

```diff
@@ -165,6 +165,83 @@ func TestFlipCassandraClusterUpdateSeedListStatusScaleDC2(t *testing.T) {
 	assert.Equal(true, reflect.DeepEqual(b, status.SeedList))
 }
 
+func TestFlipCassandraClusterUpdateSeedListStatusScaleDC2ManualSeedList(t *testing.T) {
+	assert := assert.New(t)
+
+	_, cc := helperInitCluster(t, "cassandracluster-2DC-ManualSeedlist.yaml")
+	//Allow Update SeedList
+	cc.Spec.AutoUpdateSeedList = false
+
+	assert.Equal(api.ClusterPhaseInitial, cc.Status.CassandraRackStatus["dc1-rack1"].CassandraLastAction.Name)
+	assert.Equal(api.ClusterPhaseInitial, cc.Status.CassandraRackStatus["dc1-rack2"].CassandraLastAction.Name)
+	assert.Equal(api.ClusterPhaseInitial, cc.Status.CassandraRackStatus["dc2-rack1"].CassandraLastAction.Name)
+	assert.Equal(3, len(cc.Status.CassandraRackStatus))
+
+	//cc.Status.SeedList = cc.InitSeedList()
+
+	var a = []string{
```

Comment:
Let's use a variable name that has a meaning

Reply:
ok